shared/domain/src/main/java/com/ivy/domain/features/IvyFeatures.kt

```diff
@@ -16,10 +16,17 @@ class IvyFeatures @Inject constructor() : Features {
         description = "Smaller transactions cards" +
             " so you can fit more on your screen."
     )
+    override val sortCategoriesAlphabetically = BoolFeature(
```

Comment:
Nice 💯 Can you delete the above features for "homeV2" and "compactTransactions" because they were dummy ones